axelor-account/src/main/resources/views/MoveLine.xml

```diff
@@ -803,6 +803,11 @@
     <action name="action-method-account-moveline-usher-process"/>
   </action-group>
 
+  <action-group name="action-move-line-group-analytic-move-line-onchange">
+    <action name="action-move-line-method-analytic-move-line-onchange"/>
+    <action name="action-analytic-move-line-group-analytic-move-line-list-onchange"/>
+  </action-group>
```

Comment:
Can't this action group be managed in `MoveLineGroupService` ?

Reply:
No,
this action-group : action-analytic-move-line-group-analytic-move-line-list-onchange
Call an action-attrs and an action-condition